frontend-react/src/TelemetryService.ts

```diff
@@ -45,6 +46,15 @@ const createTelemetryService = () => {
         });
     };
 
+    // Add user email as user_AuthenticatedId to all tracking events
+    OKTA_AUTH.authStateManager.subscribe((authState) => {
+        const { email } = authState?.idToken?.claims || {};
```

Comment:
Yes. The optional properties could cause an undefined which would throw on trying to unpack it.

Reply:
I see, I guess if that's true, then we wouldn't want to use destructuring then so something like:

```
const email = authState?.idToken?.claims?.email
```

so that if `email` is `undefined` that's fine for our usecase since it's falsy.

Reply:
Either's fine in my opinion since shallow destructuring from an empty object will always give `undefined` 🤷

Reply:
It's a potato-e/potato-h scenario here 😆. Since we're only accessing one thing you can argue to just reference it directly while my usual instinct is to unpack properties in objects as I can go back to it later and unpack more properties as needed.